### PR TITLE
Mention JupyterLab Desktop in the installation section

### PR DESCRIPTION
**Pull request by krassowski**
## References

Noticed when answering https://github.com/jupyterlab/jupyterlab/issues/14589#issuecomment-2033907051

## Code changes

None

## User-facing changes

None

## Backwards-incompatible changes

None

### CODE REVIEW

**Comment by krassowski** [comment]
CC @mbektas.

**Review by fcollonval — APPROVED** [review]
Thanks

**Comment by mbektas** [comment]
LGTM @krassowski . Thanks!

File: docs/source/getting_started/installation.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@
 Installation
 ============
 
-JupyterLab can be installed using ``conda``, ``mamba``, ``pip``, ``pipenv`` or ``docker``.
+JupyterLab can be installed as a terminal-launched application accessible via an web browser (default), or as desktop application which is running in its own window and can be opened by clicking on a desktop shortcut (`JupyterLab Desktop <https://github.com/jupyterlab/jupyterlab-desktop>`__). This page describes installation of the default (terminal-launched) JupyterLab application by using ``conda``, ``mamba``, ``pip``, ``pipenv`` or ``docker`` and assumes basic knowledge of the terminal. For JupyterLab Desktop instructions see the `Installation section <https://github.com/jupyterlab/jupyterlab-desktop#installation>`__ in the JupyterLab Desktop repository.
```
Comment (krassowski) [inline docs/source/getting_started/installation.rst:9]:
```suggestion
JupyterLab can be installed as a terminal-launched application accessible via a web browser (default), or as a desktop application which is running in its own window and can be opened by clicking on a desktop shortcut (`JupyterLab Desktop <https://github.com/jupyterlab/jupyterlab-desktop>`__). This page describes installation of the default (terminal-launched) JupyterLab application using ``conda``, ``mamba``, ``pip``, ``pipenv`` or ``docker`` and assumes basic knowledge of the terminal. For JupyterLab Desktop instructions see the `Installation section <https://github.com/jupyterlab/jupyterlab-desktop#installation>`__ in the JupyterLab Desktop repository.
```

**Pull request merged**